bin/bexley/make-bexley-ww-postcode-db

```diff
@@ -246,20 +257,10 @@ while( my $row = $csv->getline($fh) ) {
             $row_h{SAO_END_NUMBER},
             $row_h{SAO_END_SUFFIX},
             $row_h{SAO_TEXT},
+
+            $blpu_class,
         );
     }
 }
 
-# It is possible for UPRNs in child_uprns to not occur in the postcodes table
```

Comment:
Turns out that the P/PP check fixes this issue as well, so have deleted this portion.